### ISSUE TITLE
Concatinate split log line

### ISSUE THREAD

**Issue by cosmo0920** [fluent-plugins-nursery/fluent-plugin-concat#46]
This issue is reserved as *a future work*.

#### Problem

Docker split huge log line into several logs.

#### Steps to replicate

```
$ docker run --log-driver=fluentd -it perl perl -e 'print "START" ;print "X" x 24000; print "END\n"'
```

#### Future Expected Behavior

Incoming the first split message contains `"partial_message":"true"` and the second message does not contain it.
And then, they will be concatenated into one line.

Currently, this specification is *work in progress*.

#### Your environment

* OS version
 Debian GNU/Linux Stretch
* paste result of ``fluentd --version`` or ``td-agent --version``
% bundle exec fluentd --version
fluentd 1.1.0
* plugin version
  * HEAD of master (future work)

**Comment by okkez** [comment]
The feature has not been released yet. 18.05.0-ce.

**Comment by okkez** [comment]
Released v2.3.0.
